package.json

```diff
@@ -12,6 +12,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-pro-sidebar": "^0.7.1",
```

Comment:
@Vaishnavi108 you should avoid using packages, for such things, this side bar you should try creating your own!

Reply:
I used the package thinking..it is more efficient.. and easier to code as well..can i know why you don't recommend using it ?

Reply:
You are a beginner @Vaishnavi108, so try to do things by your own first, instead of using library packages for most things, it will help you to understand better